### DISCUSSION TITLE
GPD pocket 3 - Which chromeOS recovery is used for intel pentium N6000 ?

### DISCUSSION THREAD

**Discussion by efkosk** [sebanc/brunch#1827] [Q&A]
Hi, I'm trying to install chromeOS but without success. When booting brunch, the computer reboots.
I have a GPD Pocket 3 (Intel pentium N6000) and I downloaded "Volteer". Should I download another version? If so, which one?
Thank you

**Comment by spekulant2** [comment]
install versions hatch
